docs/_optimist/guided_tour/step01/index.de.md

```diff
@@ -29,9 +29,9 @@ ein neues Passwort, schreiben Sie uns bitte eine E-Mail
 an <support@gec.io>.
 
 Hierzu wechseln wir im Browser auf folgende
-URL: <https://dashboard.optimist.innovo.cloud/>
+URL: <https://.optimist.gec.io/>
```

Comment:
```suggestion
URL: <https://dashboard.optimist.gec.io>
```

Reply:
Thanks, I think it should just be "<https://.optimist.gec.io/>" without the dashboard part. Removed the "." too

As per the mail:
Ebenfalls können sie das Dashboard der Optimist Plattform nun direkt unter [optimist.gec.io] erreichen.